build/config/linux/BUILD.gn

```diff
@@ -15,6 +15,9 @@ group("linux") {
 # is applied to all targets. It is here to separate out the logic that is
 # Linux-only. This is not applied to Android, but is applied to ChromeOS.
 config("compiler") {
+ if(is_linux) {
+    defines = ["__unused=__attribute__((unused))"]
```

Comment:
Just a neat.
Since `__unused` only be used in react-native source, how about adding the defines as a config in react-native build config, e.g. [rn_cxx_component](https://github.com/Kudo/react-native-skia/blob/main/build/secondary/react-native/common.gni)

Reply:
Good point. 
Actually I am new to Chromium building ecosystem and I just referred other gn files :-)
While addressing this cmment I found out that `__unused` is also indirectly used in ReactSkia for [source_set("ReactSkia")](https://github.com/Kudo/react-native-skia/blob/main/ReactSkia/BUILD.gn) and [executable("ReactSkiaApp")](https://github.com/Kudo/react-native-skia/blob/main/ReactSkia/BUILD.gn), because they include some headerfiles from react-native which uses` __unused`  . So I did following

1. defined `react_native_config` config in [ReactCommon Build.gn]( https://github.com/Kudo/react-native-skia/blob/main/build/secondary/react-native/ReactCommon/BUILD.gn)
```
config("react_native_config") {
  if(is_linux) {
    defines = ["__unused=__attribute__((unused))"]
  }
```
2.  Added following config in [rn_cxx_component](https://github.com/Kudo/react-native-skia/blob/main/build/secondary/react-native/common.gni),  [source_set("ReactSkia")](https://github.com/Kudo/react-native-skia/blob/main/ReactSkia/BUILD.gn) and [executable("ReactSkiaApp")](https://github.com/Kudo/react-native-skia/blob/main/ReactSkia/BUILD.gn)

`configs += [ "//react-native/ReactCommon:react_native_config"]`

if you are fine with this proposal, I will submit the change and update the pull request.

Reply:
The idea is pretty great that we could reuse the common config in the future.
You know GN really well.
Please update with the change and I will merge then.
Thank you.

Reply:
Done. Also updated the Readme file. Kindly have a look and do the needful.

Reply:
Nice. Thanks for your kindly update.
I am going to merge this.